Cargo.toml

```diff
@@ -1,2 +1,2 @@
 [workspace]
-members = ["libtransact"]
+members = ["libtransact", "transact_derive"]
```

Comment:
Commit message is wrong - "Add transact-derive"

Also, add some context in the commit message describing what the library includes, as of this commit.